src/containers/userInfo.js

```diff
@@ -2,38 +2,45 @@ import React, { Component } from "react";
 import axios from "axios";
 import ResponsiveLayout from "../components/ResponsiveLayout";
 
-let state = {
-  email: ""
-};
+
 export default WrappedComponent =>
   class userInfo extends Component {
+    constructor(props) {
+      super(props);
+      this.state = {
+        id: "",
+        email: ""
+      };
+    }
+
     componentDidMount() {
       this.getUser();
     }
 
     onResponseInfo = response => {
-      const { email } = response.data;
-      state = { email };
-      this.forceUpdate();
+      this.setState({id: response.data.id, email: response.data.email});
+      console.log(this.state);
```

Comment:
I think we can take out this `console.log` now

Reply:
Forgot, my bad